drivers/bch/Kconfig

```diff
@@ -34,4 +34,11 @@ config BCH_DEVICE_READONLY
 	---help---
 		Set bch devices read-only
 
+config BCH_FORCE_INDIRECT
+	bool "Force indirect transfers in BCH"
+	default n
+	---help---
+		Force all data transfers in BCH to be indirect,
+		using an intermediate buffer.
```

Comment:
@jpaali please include the comment you put in the commit log here in the help, it will instruct people to understand better when to enable it

Reply:
done